scripts/lp.js

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
whole script should execute on document ready, and probably in a self executing fn

https://learn.jquery.com/using-jquery-core/document-ready/
http://markdalgleish.com/2011/03/self-executing-anonymous-functions/